app/tweet_collection/job.py

```diff
@@ -213,15 +223,27 @@ def parse_response(self, response):
             "width": m.get("width"),
         } for m in media]
 
-
-
-
         #print("MEDIA:")
         #pprint(media_records)
 
+        if not response.data:
```

Comment:
FYI - @LucyJulian98 I am finding situations where we reach a page / response that has no data.